edge/rules.v4

```diff
@@ -0,0 +1,37 @@
+# Generated by iptables-save v1.6.0 on Tue Jun  5 16:29:15 2018
+*nat
+:PREROUTING ACCEPT [0:0]
+:INPUT ACCEPT [0:0]
+:OUTPUT ACCEPT [0:0]
+:POSTROUTING ACCEPT [0:0]
+-A POSTROUTING -o eth0 -j MASQUERADE
+COMMIT
+# Completed on Tue Jun  5 16:29:15 2018
+# Generated by iptables-save v1.6.0 on Tue Jun  5 16:29:15 2018
+*filter
+:INPUT DROP [0:0]
+:FORWARD DROP [0:0]
+:OUTPUT ACCEPT [0:0]
+-A INPUT -i lo -j ACCEPT
+-A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
+-A INPUT -i eth0 -p udp -m udp --dport 67 -j ACCEPT
+-A INPUT -i eth0 -p tcp -m tcp --dport 22 -j ACCEPT
+-A INPUT -i eth0 -p udp -m udp --dport 25 -j ACCEPT
+-A INPUT -i eth0 -p tcp -m tcp --dport 25 -j ACCEPT
+-A INPUT -i eth0 -p udp -m udp --dport 53 -j ACCEPT
+-A INPUT -i eth0 -p tcp -m tcp --dport 53 -j ACCEPT
+-A INPUT -i eth0 -p icmp -j ACCEPT
+-A INPUT -i wlan0 -p icmp -m icmp --icmp-type 8 -j ACCEPT
+-A INPUT -i wlan0 -p icmp -m icmp --icmp-type 0 -j ACCEPT
+-A FORWARD -i wlan0 -o eth0 -j ACCEPT
+-A FORWARD -i eth0 -o wlan0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
```

Comment:
Allowing packets to be forwarded from wlan0 and outgoing if related or established for wireless access point.